docs/integracion-continua.md

```diff
@@ -0,0 +1,15 @@
+# Integración Continua     
+Para la búsqueda me he centrado en aquellos sistemas de integración continua que se puedan usar con el leguaje del proyecto, que estén integrandos en la nube y que se puedan usar en GitHub. Dentro de estos he seleccionado para usar en el proyecto aquellos que sean gratuitos, que no se excedan en los permisos para vincularlos con GitHub y que no demanden ciertos datos personales como puede ser el número de tarjeta.   
```

Comment:
Pone *leguaje* e *integrandos*. Esto no es una tesis doctoral, pero el mismo IDE os tiene que avisar de esto si le enchufáis el plugin adecuado. Por favor, hazlo.

Reply:
No me había dado cuenta y eso que lo he estado mirando un buen rato en busca del fallo :joy:

Reply:
De verdad, el vim seguro que tiene un fly-spell-check. El emacs lo tiene. Si no lo tiene el vim, ¡pásate a emacs!

Reply:
Justo es una de las cosas que estoy mirando y parece que si lo tiene,  así que por ahora me quedo en vim :satisfied: